packages/audit/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@pnpm/audit",
+  "version": "0.0.0",
+  "description": "Audit a lockfile",
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "lint": "tslint -c tslint.json src/**/*.ts test/**/*.ts",
+    "tsc": "rimraf lib && tsc",
+    "test": "pnpm run tsc && pnpm run lint && ts-node test",
+    "prepublishOnly": "pnpm run tsc"
+  },
+  "repository": "https://github.com/pnpm/pnpm/blob/master/packages/audit",
+  "keywords": [
+    "pnpm",
+    "audit"
+  ],
+  "author": "Zoltan Kochan <z@kochan.io> (https://www.kochan.io/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/pnpm/pnpm/issues"
+  },
+  "homepage": "https://pnpm.js.org",
+  "devDependencies": {
+    "@pnpm/audit": "link:",
+    "@pnpm/lockfile-file": "workspace:3.0.1",
+    "@types/node-fetch": "2.5.3",
```

Comment:
I&nbsp;believe&nbsp;that this&nbsp;is no&nbsp;longer&nbsp;necessary.

Reply:
yes, thanks